src/doc/intro.md

```diff
@@ -213,7 +213,7 @@ Segmentation fault (core dumped)
 
 A crash! (Note that this is actually system-dependent. Because referring to an
 invalid reference is undefined behavior, the compiler can do anything,
-including the right thing!) Even though we compiled with flags to give us as
+including the right thing!). Even though we compiled with flags to give us as
```

Comment:
This is not needed. Note that on 214, "A crash!" has an exclamation mark to end the sentence, and here, inside the `()`s, there's also an exclamation mark to end the sentence.